src/Apps/Sell/Routes/DimensionsRoute.tsx

```diff
@@ -69,33 +75,54 @@ export const DimensionsRoute: React.FC<DimensionsRouteProps> = props => {
           <Text mb={2} variant="xl">
             Artwork dimensions
           </Text>
-          <RadioGroup
-            onSelect={option => setFieldValue("dimensionsMetric", option)}
-            defaultValue={values.dimensionsMetric}
-          >
-            <BorderedRadio value="in" label="in" />
-            <BorderedRadio value="cm" label="cm" />
-          </RadioGroup>
-          <Spacer y={2} />
-          <Join separator={<Spacer y={2} />}>
-            <Input
-              onChange={handleChange}
-              name="width"
-              title="Width"
-              defaultValue={values.width}
-            ></Input>
-            <Input
-              onChange={handleChange}
-              name="height"
-              title="Height"
-              defaultValue={values.height}
-            ></Input>
-            <Input
-              onChange={handleChange}
-              name="depth"
-              title="Depth"
-              defaultValue={values.depth}
-            ></Input>
+          <Join separator={<Spacer y={4} />}>
```

Comment:
Just to make it easier to read :) 
```suggestion

          <Join separator={<Spacer y={4} />}>
```